test/use-cases/demographics.integration.js

```diff
@@ -0,0 +1,231 @@
+/* global describe,before,it */
+
+'use strict';
+
+/* eslint max-len: 0 */
+
+process.env.NODE_ENV = 'test';
+
+const _ = require('lodash');
+const chai = require('chai');
+const moment = require('moment');
+
+const Generator = require('../util/generator');
+const SharedIntegration = require('../util/shared-integration');
+const RRSuperTest = require('../util/rr-super-test');
+const exampleSurveys = require('../fixtures/example/survey');
+const config = require('../../config');
+
+const expect = chai.expect;
+
+// TODO: eventually assign these to the key of answerValueType?
+// NOTE: see `/models/dao/demographics.dao.js` ._castAnswerValueByType()
+const getAnswerValue = (question) => {
+    if (question.type === 'text') {
+        return question.answer.textValue;
+    } else if (question.type === 'integer') {
+        return parseInt(question.answer.integerValue, 10);
+    } else if (question.type === 'zip') {
+        return question.answer.textValue;
+    } else if (question.type === 'year') {
+        return question.answer.yearValue;
+    } else if (question.type === 'bool') {
+        return question.answer.boolValue;
+    } else if (question.type === 'date') {
+        return question.answer.dateValue;
+    }
+    // // FIXME: only returns a true value... need to join with questionChoice
+    // else if(question.type === 'choice') {
+    //     return question.answer.choice;
+    // }
+    // // // FIXME will always be null... need to join with questionChoice
+    // else if(question.type === 'choices') {
+    //     return question.answer.choices;
+    // }
+
+    return question.answer.textValue;
+};
+
+describe('demographics', function ageCohort() {
+    const rrSuperTest = new RRSuperTest();
+    const generator = new Generator();
+    const shared = new SharedIntegration(rrSuperTest);
+
+    let expectedDemographics = [];
+
+    before(shared.setUpFn());
+
+    it('login as super user', shared.loginFn(config.superUser));
+
```

Comment:
If you add a survey before you are adding profile survey the tests will fail because of code defects.
    it('create a survey', tests.createSurveyFn());